RayTraceAntiXray/src/main/java/com/vanillage/raytraceantixray/RayTraceAntiXray.java

```diff
@@ -59,7 +60,7 @@ public void onEnable() {
         executorService = Executors.newFixedThreadPool(Math.max(getConfig().getInt("settings.anti-xray.ray-trace-threads"), 1));
         timer = new Timer(true);
         timer.schedule(new RayTraceTimerTask(this), 0L, Math.max(getConfig().getLong("settings.anti-xray.ms-per-ray-trace-tick"), 1L));
-        new UpdateBukkitRunnable(this).runTaskTimer(this, 0L, Math.max(getConfig().getLong("settings.anti-xray.update-ticks"), 1L));
+        SchedulerUtil.runTaskTimer(this, new UpdateBukkitRunnable(this), 0L, Math.max(getConfig().getLong("settings.anti-xray.update-ticks"), 1L));
```

Comment:
I think for Paper we can keep this and for Folia we should replace it by `EntityScheduler`s per player on join.